CMakeLists.txt

```diff
@@ -44,13 +44,14 @@ set(QUIC_FBCODE_ROOT ${CMAKE_CURRENT_SOURCE_DIR})
 find_package(Boost 1.62
   REQUIRED COMPONENTS
   iostreams
-  system
   thread
   filesystem
   regex
   context
   date_time
   program_options
+  OPTIONAL_COMPONENTS
+  system
```

Comment:
https://github.com/facebook/mvfst/blob/3603e7fc32de7203b5fc227b8de3013cac676150/CMakeLists.txt#L6

CMake 3.11 introduced `OPTIONAL_COMPONENTS` but project allows 3.10 so will need some more work.

Easiest options are increasing CMake to 3.11 or increasing Boost to 1.69. Otherwise, may have to run `find_package` twice.